modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/forms-field-sidebar/components/FormsFieldSidebar.es.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import classNames from 'classnames';
+import React, {useContext} from 'react';
+
+import {FormsSidebarPluginContext} from '../../../components/sidebar/MultiPanelSidebarFormsProxy.es';
+import {FieldsSidebar} from '../../fields-sidebar/components/FieldsSidebar.es';
+
+const sortFieldTypes = (fieldTypes) =>
+	fieldTypes.sort(({displayOrder: a}, {displayOrder: b}) => a - b);
+
+export const FormsFieldSidebar = ({title}) => {
+	const {
+		activePage,
+		dataProviderInstanceParameterSettingsURL,
+		dataProviderInstancesURL,
+		defaultLanguageId,
+		dispatch,
+		editingLanguageId,
+		fieldTypes,
+		focusedCustomObjectField,
+		focusedField,
+		functionsMetadata,
+		functionsURL,
+		pages,
+		rules,
+	} = useContext(FormsSidebarPluginContext);
+
+	const config = {
+		allowFieldSets: true,
+		allowMultiplePages: false,
+		allowNestedFields: false,
+		allowRules: true,
+		allowSuccessPage: false,
+		disabledProperties: [],
+		disabledTabs: [],
+		ruleSettings: {
+			dataProviderInstanceParameterSettingsURL,
+			dataProviderInstancesURL,
+			functionsMetadata,
+			functionsURL,
+		},
+		unimplementedProperties: [
+			'allowGuestUsers',
+			'fieldNamespace',
+			'readOnly',
+			'visibilityExpression',
+		],
+	};
+
+	const hasFocusedCustomObjectField = () => {
+		return false;
+	};
+
+	const hasFocusedField = Object.keys(focusedField).length > 0;
+
+	const fieldTypesSorted = sortFieldTypes(
+		fieldTypes.filter(({group}) => group === 'basic')
+	);
+
+	const dataLayout = {
+		dataLayoutFields: [],
+		dataLayoutPages: [],
+		dataRules: rules,
+		name: '',
+		paginationMode: 'single-page',
+	};
+
+	return (
+		<FieldsSidebar
+			classNames={classNames}
+			config={config}
```

Comment:
Just to avoid declaring a variable 🙂.

```suggestion
			config={{
				allowFieldSets: true,
				allowMultiplePages: false,
				allowNestedFields: false,
				allowRules: true,
				allowSuccessPage: false,
				disabledProperties: [],
				disabledTabs: [],
				ruleSettings: {
					dataProviderInstanceParameterSettingsURL,
					dataProviderInstancesURL,
					functionsMetadata,
					functionsURL,
				},
				unimplementedProperties: [
					'allowGuestUsers',
					'fieldNamespace',
					'readOnly',
					'visibilityExpression',
				],
			}}
```

Reply:
What about
```js
const FIELD_SIDEBAR_CONFIG = {
...
};
```
?

Reply:
@matuzalemsteles, I think it may decrease the code readability due to the complexity camming  from the `config` variable that will be added to the component declaration. Don't you think?

@diegonvs I think this suggestion won't fit in this case, because this naming pattern belongs to constants  and the `ruleSettings` is dynamic.

Reply:
@miltonmc yeah, makes sense. We can keep it in a variable for now.